tests/View/ViewBladeCompilerTest.php

```diff
@@ -177,16 +177,6 @@ public function testDontIncludeEmptyPath()
         $compiler->compile();
     }
 
-    public function testDontIncludeNullPath()
-    {
-        $compiler = new BladeCompiler($files = $this->getFiles(), __DIR__);
-        $files->shouldReceive('get')->once()->with(null)->andReturn('Hello World');
-        $files->shouldReceive('exists')->once()->with(__DIR__)->andReturn(true);
-        $files->shouldReceive('put')->once()->with(__DIR__.'/'.sha1(null).'.php', 'Hello World');
-        $compiler->setPath(null);
```

Comment:
Please find all places in the framework that might call this will null, and fix them.

Reply:
There is at least one, in the service provider.

Reply:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/26096713/115970209-41c7c300-a541-11eb-9d28-490fe2dc1aac.png">

Reply:
It's OK.

Reply:
> There is at least one, in the service provider.

Sorry, didn't see this comment. Where exactly it is? Even using normal search, without any PhpStorm magic, I can't find it.